frontend/src/metabase/lib/expressions/config.js

```diff
@@ -388,7 +388,8 @@ export const EXPRESSION_OPERATORS = new Set(["+", "-", "*", "/"]);
 export const FILTER_OPERATORS = new Set(["!=", "<=", ">=", "<", ">", "="]);
 
 export const BOOLEAN_UNARY_OPERATORS = new Set(["not"]);
-export const BOOLEAN_BINARY_OPERATORS = new Set(["and", "or"]);
+export const LOGICAL_AND_OPERATOR = new Set(["and"]);
+export const LOGICAL_OR_OPERATOR = new Set(["or"]);
```

Comment:
Maybe these shouldn't be sets? I don't know when we'd add aliases for and/or.

Reply:
I think this is the way Chevrotain configures the lexer (weird indeed). I try to restrict the changes and just follow the existing patterns so far.